website/docs/concepts/nodes-networks.md

```diff
@@ -0,0 +1,111 @@
+import NetworkPng from '@site/static/img/network.png';
+import NetworkLayersPng from '@site/static/img/network-layers.png';
+
+
+Ethereum is a decentralized **network** of **nodes** that communicate peer-to-peer:
+
+<img style={{width: 100 + '%', margin: 'auto', display: 'block', maxWidth: 461 + 'px'}} src={NetworkPng} /> 
+
+
+## Nodes
+
+An Ethereum **node** is a running instance of Ethereum client software. There are two primary types of nodes in Ethereum: **execution nodes** and **beacon nodes**. Colloquially, a "node" refers to an execution node and beacon node working together.
+
+When users stake 32 ETH to participate in Ethereum's proof-of-stake consensus mechanism, they use validator client software like Prysm to stake their ETH and run a **validator**. Validators are not nodes - a validator can be thought of as an extension of a beacon node that manages validator keys and duties. Validators depend on beacon nodes, which depend on execution nodes:
+
+<br />
+
+<img style={{width: 100 + '%', margin: 'auto', display: 'block', maxWidth: 651 + 'px'}} src={ClientStackPng} /> 
+
+<br />
+
+<table>
+    <tr>
+        <th style={{minWidth: 170 + 'px'}}>Component</th> 
+        <th>Description</th>
+    </tr>
+    <tr>
+      <td><strong>Ethereum node</strong><br />aka "Node"</td>
+      <td>An Ethereum node is an <strong>execution node</strong> and <strong>beacon node</strong> working together. Ethereum nodes communicate peer-to-peer to secure the Ethereum network, and require both <strong>execution-layer client software</strong> and <strong>consensus-layer client software</strong>.</td>
+    </tr> 
+    <tr>
+      <td><strong>Execution node</strong></td>
+      <td>Execution nodes use execution client software to process transactions and smart contracts in Ethereum's <strong>execution layer</strong>. Nethermind, Besu, and Geth are execution clients.<br /> <br />Execution nodes talk to each other, and to beacon nodes.</td>
+    </tr>
+    <tr>
+      <td><strong>Beacon node</strong></td>
+      <td>Beacon nodes use beacon node client software to coordinate Ethereum's proof-of-stake consensus mechanism in Ethereum's <strong>consensus layer</strong>. Prysm, Teku, Lighthouse, and Nimbus are consensus clients that contain both beacon node and validator client software. <br /> <br />Beacon nodes talk to each other, to execution nodes, and to validators.</td>
+    </tr>
+    <tr>
+      <td><strong>Validator</strong></td>
+      <td>Validators let people stake 32 ETH as collateral within Ethereum's <strong>consensus layer</strong>. Validators are responsible for proposing blocks within Ethereum's proof-of-stake consensus mechanism, and will fully replace proof-of-stake miners after The Merge. <br /> <br />Any given validator talks only to the one beacon node that it's connected to. A validator's beacon node tells the validator what work to do, and broadcasts the validator's work to the Ethereum network as the validator performs its duties.</td>
```

Comment:
```suggestion
      <td>Validator clients are specialized software that let people stake 32 ETH as collateral within Ethereum's <strong>consensus layer</strong>. Validators are responsible for proposing blocks within Ethereum's proof-of-stake consensus mechanism, and will fully replace proof-of-stake miners after The Merge. <br /> <br />Any given validator talks only to the one beacon node that it's connected to. A validator's beacon node tells the validator what work to do, and broadcasts the validator's work to the Ethereum network as the validator performs its duties.</td>
```
Does "the merge" need a hyperlink as well? Maybe something to note, as im not sure if we've ever mentioned it in this document